core/connection.js

```diff
@@ -673,7 +673,7 @@ Blockly.Connection.prototype.onCheckChanged_ = function() {
 Blockly.Connection.prototype.setCheck = function(check) {
   if (check) {
     // Ensure that check is in an array.
-    if (!goog.isArray(check)) {
+    if (!goog.isArrayLike(check)) {
```

Comment:
`Array.isArray`?

Reply:
In google-closure-library@20200830.0.0 the goog.isArray removed, i just found this [goog.isArrayLike](https://google.github.io/closure-library/api/goog.html#isArrayLike)

Reply:
Why not use the actual `Array.isArray`?

Reply:
`goog.isArrayLike` accept string, `Array.isArray` is better!!